content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+tags:
+  - hpe-greenlake
+  - terraform
+  - bare-metal-as-a-service
+  - hpegl
+  - BMaaS
+---
+# Introduction
+
+The HPE GreenLake for Private Cloud Enterprise: bare metal (BMaaS) service offering enables you to create dedicated compute instances deployed on a physical IT infrastructure facilitating on-demand scalability, convenience, and agility as a cloud service. 
+Using BMaaS, you can create compute-instances provisioned with specific operating systems, network connections, one or more public SSH keys, and optional network-attached storage volumes.
+The service can be accessed via GUI as well as via public APIs, enabling developers to use an Infrastructure-as-Code tool to build, change, and manage infrastructure in a consistent and repeatable way.
+
+## HPE GreenLake Terraform Provider
+
+The HPE GreenLake Terraform provider **hpegl** by HPE GreenLake provides Infrastructure-as-Code support for HPE GreenLake Cloud Services.
+Using the **hpegl** Terraform provider, you can automate the management of your infrastructure. You can provision OS on bare metal, spin virtual machines and bring up a Kubernetes cluster, starting right from bare metal all the way up in the stack to desired configurations and applications.
+
+In this blog post, I will walk you through the steps required to use the HPE GreenLake Terraform Provider to deploy and further manage bare metal compute instances.
+
+# Preparing for Infrastructure-as-Code implementation
+
+## Setting up API Client for access
+
+You need an API client to authenticate against the HPE GreenLake.
+
+Follow the below steps for API Client creation:
+
+1. From the HPE GreenLake platform, launch the HPE GreenLake Central console for the appropriate tenant. Under the settings icon on the tenant Dashboard page, select the **User Management** option. 
+
+![User Management](/img/greenlake_console_usermanagement.png "User Management")
+
+2. Under the API Clients tab, click on **Create API Client**.
+
+![Create API Client](/img/greenlake_console_createapiclient.png "Create API Client")
+
+3. Enter a **Name** (mandatory field) and **Description** (optional) for the API client, and click on **Create** button.
+
+![Create API Client](/img/greenlake_console_createapiclient_form.png "Create API Client")
+
+4. Ensure you make a note of the **Issuer**, **Client ID**, and **Client Secret** before clicking on the **Close** button. These details will be exported as environment variables in the next section.
+
+![API Client Created](/img/greenlake_conolse_apiclient_created.png "API Client Created")
+
+5. In the API Clients page, select the newly created client, and click on **Create Assignment** button.
+
+
+6. Assign the roles **BMAAS Access Viewer** and **BMAAS Access Project Contributor** on the **Space: Default.**
+
+![BMaaS Roles](/img/greenlake_console_createbmaasassignment.png "BMaaS Roles")
+
+The API Client is now ready to be used to run the Terraform resources.
+
+## Select the Compute Group ID
```

Comment:
When it comes to initial capitalization, it's recommended to only use this for proper nouns (names of products/programs) and in instances where you would see it on a screen shot this way. I looked below to see what was in your screen shot and I do see "Compute group", but that's because it's a title of the command string and the first word is "Compute". This indicates to me that "Compute Group" is not, in general, a proper noun. So my recommendations now are based on that observation.

Here, for this header, I would change it to "Select the compute group ID"